.github/scripts/trymerge.py

```diff
@@ -351,6 +355,7 @@ def merge_into(self, repo: GitRepo, dry_run: bool = False) -> None:
         if not self.is_ghstack_pr():
             msg = self.get_title() + "\n\n" + self.get_body()
             msg += f"\nPull Request resolved: {self.get_pr_url()}\n"
+            msg += f"Reviewed by: {', '.join(self.get_approved_by())}\n"
```

Comment:
Should this be "Approved by:" instead?

Reply:
+1 to Approved by

Reply:
I don't know. But as we are using "Reviewed by" right now, perhaps we should stick to it

Reply:
Tangent question as well, would it be useful to have a `Merged with GHF` kind of breadcrumb in the commit message so that internal tooling can be aware of when we're looking at a GHF commit?

Reply:
Changing the message to "Approved by"